src/views/AccountDrefApplications/DrefTableActions/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { useContext, useMemo } from 'react';
+import { generatePath } from 'react-router-dom';
+import {
+    AddLineIcon,
+    CaseManagementIcon,
+    CheckLineIcon,
+    DocumentPdfLineIcon,
+    DownloadLineIcon,
+    PencilLineIcon,
+    ShareLineIcon,
+} from '@ifrc-go/icons';
+
+import DropdownMenuItem from '#components/DropdownMenuItem';
+import TableActions from '#components/Table/TableActions';
+import Link from '#components/Link';
+import useTranslation from '#hooks/useTranslation';
+import RouteContext from '#contexts/route';
+import { components } from '#generated/types';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+const DREF_STATUS_COMPLETED = 1;
+const DREF_STATUS_IN_PROGRESS = 0;
```

Comment:
Let's move this to utils/constants and also use `satisfies` that the enum value is valid.